.github/workflows/tests.yml

```
@@ -15,6 +15,10 @@ jobs:
         php: [ 7.2, 7.3, 7.4, '8.0', 8.1 ]
```

@driesvints like so?

```suggestion
        php: [ 7.2, 7.3, 7.4, '8.0', 8.1, 8.2 ]
```